package.json

```
@@ -51,7 +51,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">= 8.11",
+    "node": ">= 8.11 <12.0.0",
```

does this work for syntax?
```suggestion
    "node": ">= 8.11 < 12.0.0",
```

---

Seems to show the same error 🤷‍♂️